src/rgw/rgw_common.h

```diff
@@ -1693,6 +1693,7 @@ struct req_state {
   bool enable_usage_log;
   uint8_t defer_to_bucket_acls;
   uint32_t perm_mask;
+  string subuser_name;
```

Comment:
But currently we've [initialized subuser without checking whether the subuser exists](https://github.com/ceph/ceph/blob/master/src/rgw/rgw_auth.h#L443),
and [get_perm_mask() is using the no-verified subuser_name](https://github.com/ceph/ceph/blob/master/src/rgw/rgw_auth.h#L453)

Plz review repushed draft to parse `perm_mask` and `subuser` once.